docs/docs/clients/client-side/ios.md

```diff
@@ -150,6 +150,28 @@ func determineAppConfiguration() async throws {
 }
 ```
 
+### Providing Default Flags
+
+You can define default flag values when initialising the SDK. This ensures that your application works as intended in
+the event that it cannot receive a response from our API.
+
+```swift
+// set default flags
+Flagsmith.shared.defaultFlags = [Flag(featureName: "feature_a", enabled: false),
+                                    Flag(featureName: "font_size", intValue:12, enabled: true),
+                                    Flag(featureName: "my_name", stringValue:"Testing", enabled: true)]
+```
+
+### Cache
```

Comment:
These docs will likely need to change based on the ongoing conversation [here](https://github.com/Flagsmith/flagsmith-ios-client/pull/33#discussion_r1205661829).

Reply:
OK this is done from the other PR that is now closed https://github.com/Flagsmith/flagsmith/pull/2280